src/routes/ChatRoom/index.js

```diff
@@ -0,0 +1,4 @@
+import ChatRoom from './ChatRoom'
```

Comment:
Small thing: We most consistently make this generic in our code, i.e. `import component from './ChatRoom'`